pkg/services/star/model.go

```diff
@@ -14,8 +14,9 @@ type Star struct {
 // COMMANDS
 
 type StarDashboardCommand struct {
-	UserID      int64 `xorm:"user_id"`
-	DashboardID int64 `xorm:"dashboard_id"`
+	UserID       int64  `xorm:"user_id"`
+	DashboardID  int64  `xorm:"dashboard_id"`
+	DashboardUID string `xorm:"-"`
```

Comment:
following [this](https://github.com/grafana/grafana/pull/54340/files#r987964784) I don't think that `DashboardUID` field serves anything

Reply:
I see, get used to adding it since I was thinking to return the uid also in the post response, just saw that we don't have a returning body for post... will remove it.